File: poc-challenges/250722/src/test/java/com/tdd/domain/FibonacciTest.java
package com.tdd.domain;

import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

public class FibonacciTest {
	@Test
	void test_fibonacci() {
		int[][] cases = {{0,0}, {1, 1}, {2, 1}, {3, 2}, {4, 3}, {5, 5}, {6, 8}, {7, 13}};

		for (int[] aCase : cases) {
			assertEquals(aCase[1], Fibonacci.fib(aCase[0]));
		}
	}
}

File: poc-challenges/250907/src/test/kotlin/com/currenjin/tdd/FibonacciTest.kt
package com.currenjin.tdd

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class FibonacciTest {
    @Test
    fun fibonacci() {
        assertEquals(0, Fibonacci.fibonacci(0))
        assertEquals(1, Fibonacci.fibonacci(1))
    }
}